src/content/en/2024/third-parties.md

```diff
@@ -3,17 +3,205 @@
 title: Third Parties
 description: Third Parties chapter of the 2024 Web Almanac covering data of what third parties are used on the web and a deep dive into preventing them causing negative performance impacts.
 hero_alt: Hero image of Web Almanac characters plugging various things into a web page.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
+Yash-Vekaria_bio: TODO
+zubairshafiq_bio: TODO
```

Comment:
@zubairshafiq can you please provide a short bio about yourself?

Reply:
@tunetheweb Please use this:
Zubair Shafiq is a computer science professor at [University of California, Davis](https://www.ucdavis.edu/). His research aims to make the Internet private, secure, and safe using empirically grounded measurement and modeling methods.

Reply:
Done!